src/replica/duplication/test/replica_follower_test.cpp

```diff
@@ -104,7 +106,8 @@ class replica_follower_test : public duplication_test_base
 
     void init_nfs()
     {
-        stub->_nfs = nfs_node::create();
+        std::shared_ptr<dns_resolver> resolver = std::make_shared<dns_resolver>();
```

Comment:
Because we can infer the type from std::make_shared<xxx>, so we can use auto simply.
```suggestion
        auto resolver = std::make_shared<dns_resolver>();
```